### DISCUSSION TITLE
Unable to start 2 timebase instances in same server

### DISCUSSION THREAD

**Discussion by sujit-sarjerao-kamthe-db** [finos/TimeBase-CE#279] [Q&A]
Hello Team,

We are unable to start 2 timebase instances in same server.
It is throwing an error that, 8011 port is already in use.

could you please suggest, how i can start another instance of Timebase on port other than 8011?

Thanks

**Comment by alex-karpovich** [comment]
using script:
```
tdbserver.sh -port 8013
```

or via java 
```
java -Ddeltix.home=. -classpath ./lib/timebase-server-<version>.jar com.epam.deltix.qsrv.comm.cat.TBServerCmd -home <home> -port 8013
```
